bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Tests/Controllers/SystemRegisterControllerTest.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Altinn.Authentication.UI.Controllers;
+using Altinn.Authentication.UI.Core.SystemRegister;
+using Altinn.Authentication.UI.Mocks.SystemRegister;
+using Altinn.Authentication.UI.Mocks.Utils;
+using Altinn.Authentication.UI.Tests.Utils;
+using Altinn.Platform.Profile.Models;
+using Newtonsoft.Json;
+using System.Net;
+using System.Net.Http.Headers;
+using Xunit;
+
+namespace Altinn.Authentication.UI.Tests.Controllers;
+
+[Collection("SystemRegisterController Tests")]
+public class SystemRegisterControllerTest : IClassFixture<CustomWebApplicationFactory<SystemRegisterController>>
+{
+    private readonly CustomWebApplicationFactory<SystemRegisterController> _factory;
+    private readonly HttpClient _client;
+    private readonly ISystemRegisterService _systemRegisterService;
+
+    public SystemRegisterControllerTest(
+        CustomWebApplicationFactory<SystemRegisterController> factory)
+    {
+        _factory = factory;
+        _client = SetupUtils.GetTestClient(_factory, false);
+        _systemRegisterService = new SystemRegisterService(new SystemRegisterClientMock());
+    }
+
+    [Fact]
+    public async Task GetSystemTypeList_ReturnsListOK() 
+    {        
+        var token = PrincipalUtil.GetToken(7007, 7007, 2);
+        _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
+        HttpRequestMessage request = new(HttpMethod.Get, $"authfront/api/v1/systemregister");
+        HttpResponseMessage response = await _client.SendAsync(request, HttpCompletionOption.ResponseContentRead);
+
+        List<RegisteredSystemDTO>? list = JsonConvert.DeserializeObject<List<RegisteredSystemDTO>>(await response.Content.ReadAsStringAsync());
+        Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+        Assert.True(list is not null && list.Count > 0);        
+        Assert.True(list[0].Id is not null);
+        Assert.True(list[0].Navn is not null);
+        Assert.True(list[0].Beskrivelse is not null);
```

Comment:
Are the models defined in norwegian? Should we consider using English to be consistent?

Reply:
The model used was a temp one, since the work has not started on the Authentication component yet.
Changed propertynames to english.

Reply:
This is the DTO delivered to the Frontend. The Frontend wanted norwegian propertynames during the dev phase. Changed the names to english for consistency.